src/main/java/hw_2_technical_task/App.java

```diff
@@ -0,0 +1,12 @@
+package hw_2_technical_task;
+
+public class App {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+
+        System.out.println(x + y);
+        System.out.println(x + "" + y);
```

Comment:
THere is should be the output in one line
![Screenshot_5](https://github.com/AntBelyi/ALevel_automation/assets/66726924/cc88ce08-f443-4f25-a395-6aa3439d007f)